src/pages/books.tsx

```diff
@@ -375,7 +375,7 @@ export const getStaticProps: GetStaticProps<{
                 'Rajesh Nair',
                 'Rachelle Palmer',
             ],
-            publishedDate: 'Sept. 2023',
+            publishedDateText: 'will be published in Nov. 2023',
```

Comment:
Should we capitalize "will"?

Reply:
I can clarify with Liz, but this is the wording/capitalization that was quoted in the ticket.

Reply:
@jackyw2018 Ok Liz said to capitalize it, thanks for catching!